bndtools.core.test/bnd.bnd

```diff
@@ -0,0 +1,70 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd, ${workspace}/cnf/includes/bndtools.bnd
+-sub: *.bnd,bnd/*.bnd
+
+# These are bundles that are not required for building but for debugging when doing
+# code inspection.
+-buildpath.debug: \
+	org.eclipse.osgi,\
+	org.eclipse.core.resources,\
+	org.eclipse.e4.ui.workbench.swt,\
+	org.eclipse.search
+
+-conditionalpackage: \
+	aQute.lib.*;-split-package:=merge-first, \
+	aQute.libg.*;-split-package:=merge-first
+
+-buildpath: \
+	aQute.libg;version=snapshot,\
+	bndtools.core;version=snapshot,\
+	bndtools.builder;version=snapshot,\
+	org.junit.platform.commons;version=latest,\
+	org.junit.platform.engine;version=latest,\
+	assertj-core;version=latest,\
+	org.opentest4j;version=latest,\
+	org.apiguardian:apiguardian-api;version=latest,\
+	org.junit.jupiter.api;version=latest,\
+	org.junit.jupiter.engine;version=latest,\
+	org.junit.jupiter.params;version=latest,\
+	org.eclipse.jdt.core;version=3.16.0,\
```

Comment:
I agree with this - the reason these dependencies have dependencies is because they were added (mostly) using the quick fix processor. I think we need to consider changing the default behaviour of the quick fix processor on this point, and maybe making it configurable.

Reply:
Actually, it shouldn't be too awkward to upgrade Eclipse as the version specifiers don't give an upper bound on the version range. As long as we don't go try to backwards to support older versions of Eclipse it should still be able to find the build dependencies. I still agree that we don't need them though.

Reply:
Fixed now.